packages/common/src/types.ts

```diff
@@ -114,6 +114,21 @@ export interface IGetSurveyModelsResponse {
   stakeholder: IModel;
 }
 
+export interface IGetSurveyFormJson {
```

Comment:
I'm relatively new to typescript, but it seems odd to create interfaces based on methods and definitely not on encoding (JSON). Instead, this seems like there should be a model interface like `ISurveyForm`

Reply:
In opendata-ui, we make the distinction between a form item (item.type "Form") and its related form.json by typically calling them `formItem` and `formJson` respectively, so just carrying that over here